wastewater/dataModel.WasteWaterTreatment/WasteWaterTreatment-schema.json

```diff
@@ -0,0 +1,760 @@
+{

```

Comment:
globally it would be good to check if these properties already exist in the [database of properties](https://smartdatamodels.org/index.php/ddbb-of-properties-descriptions/ ) it could be referenced to an existing schema, or reallocated in shared schema (like this file)

Reply:
Sure! Is it possible to see an example as to how I can reference a property to an existing schema?

Reply:
[An example](https://github.com/smart-data-models/dataModel.TourismDestinations/blob/master/TouristTrip/schema.json) from tourism.
if a property is used in several data models it should be located in :
- In a shared file at subject level 'subject'-schema.json (i.e. [waternetworkmanagemet](https://github.com/smart-data-models/dataModel.WaterNetworkManagement/blob/master/WaterNetworkManagement-schema.json))
- In a shared file at domain level 'domain'-schema.json
- In a shared file at global level like [common-schema.json](https://github.com/smart-data-models/data-models/blob/master/common-schema.json)
What option should you chose? it depends on the data models that are going to share the property.

Reply:
Thanks Alberto! This is helpful. As we are also interested in connecting parameters to devices (sensors or actuators), it would make sense to therefore also refer to the Device and DeviceModel. Say for example we are interested in the dateLastCalibration property. Can we simply do the following:

"dateLastCalibration": {
         "$ref": "https://smart-data-models.github.io/dataModel.Device/Device/schema.json#/properties/dateLastCalibration"
}
or do we need to specify it in $ref with "https://smart-data-models.github.io/dataModel.Device/device-schema.json#/definitions/Device-Commons"?
But as dateLastCalibration (and many other properties) are not part of Device-Commons, I imagine this will not work?

Reply:
Both are possible, but the second option would be better.  This could mean a PR also on device repo, not a problem.